lib/index.js

```diff
@@ -12,13 +12,15 @@ function AuthPassportLocal () {
   this.context = {};
   this.pipes = pipes;
 
-  this.init = function (config, context, isDummy) {
+  this.init = function (customConfig, context, isDummy) {
     var strategy;
+    const defaultConfig = {
+      'secret': 'changeme',
```

Comment:
To be discussed with the team: it might be the right time to remove this default configuration, and to make this plugin initialization fail if no `secret` is configured in Kuzzle.

I'm not really fond of having a default salt in a password hashing module, and it may be preferable to remove the functionality until the kuzzle admin configure it properly.

What do you think?